src/lib/variants.ts

```diff
@@ -140,6 +141,25 @@ export const hasCountryCode: Filter = {
     test: (test, targeting) => (test.hasCountryName ? !!targeting.countryCode : true),
 };
 
+export const matchesCountryGroups: Filter = {
+    id: 'matchesCountryGroups',
+    test: (test, targeting): boolean => {
+        // Always True if no locations set for the test
+        if (!test.locations || test.locations.length === 0) {
+            return true;
+        }
+
+        // True or False depending on user being in one of the country groups
+        if (targeting.countryCode) {
```

Comment:
To be consistent with the early return above, what about flipping this and returning early with `false` if we don't have a countryCode?

Reply:
Yeah good shout!